renderer/brave_render_thread_observer.cc

```diff
@@ -50,6 +50,9 @@ void BraveRenderThreadObserver::OnRendererConfigurationAssociatedRequest(
   renderer_configuration_receivers_.Add(this, std::move(receiver));
 }
 
+void BraveRenderThreadObserver::SetInitialConfiguration(bool is_tor_process) {
+  is_tor_process_ = is_tor_process;
+}
 void BraveRenderThreadObserver::SetConfiguration(
```

Comment:
new line

Reply:
fixed in 2bf1be60fb4efbe59956a565c4d4d65824cae9f1